pkg/handler/handler_test.go

```diff
@@ -52,103 +52,94 @@ func (m MockBroker) Unbind(uuid.UUID, uuid.UUID) error {
 	m.called("unbind", true)
 	return m.Err
 }
+func (m MockBroker) LastOperation(uuid.UUID, *broker.LastOperationRequest) (*broker.LastOperationResponse, error) {
+	//t.Fatal("lastoperation", true)
+	state := broker.LastOperationStateInProgress
+	return &broker.LastOperationResponse{State: state, Description: ""}, nil
+}
+
+var b MockBroker
+var dahandler handler
+
+func init() {
+	b = MockBroker{Name: "testbroker"}
+	dahandler = handler{*mux.NewRouter(), b}
```

Comment:
would ```dahandler := handler{*mux.NewRouter(), b}``` work in the init? If so you could drop the variable creation above.

nit: could make the variable name ```testhandler``` to be more descriptive.